471: update the DEVELOPER_GUIDE.md instructions for running locally (#527)
IQSS/dataverse-frontend: DEVELOPER_GUIDE.md

```bash
# /dev-env/ directory

# copy the .env.example file to .env
# To test file upload, update the .env file with S3 credentials
$ cp .env.example .env

> # Installs and runs project off latest tagged container image
> # To install the latest image from the develop branch, use the 'unstable' tag
> $ ./run-env.sh <DATAVERSE_IMAGE_TAG>

Reviewer: ```suggestion
# Installs and runs project off latest tagged container image
$ ./run-env.sh unstable

# Alternatively, you can specify a tag from https://github.com/orgs/gdcc/packages/container/package/dataverse
$ ./run-env.sh unstable <DATAVERSE_IMAGE_TAG>
```

Author: Thanks @pdurbin I made those changes and added info about updating the REGISTRY in .env
